yang/frr-bfdd.yang

```diff
@@ -503,6 +503,122 @@ module frr-bfdd {
             config false;
           }
         }
+
+        list srte-sbfd-echo {
```

Comment:
this is not the place for srte objects.
either move all to frr-pathd.yang file, or leave it generic:
`list sbfd-echo` ?